atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -38,6 +38,10 @@ def run(self):
         if self.workflow.image_components is not None:
             return None
 
+        if self.workflow.builder.base_from_scratch:
+            self.log.info("from scratch can't run rpmqa")
+            return ""
```

Comment:
Why empty string?  Few lines above is used `None` as skip value.

Reply:
that was in design, but yeah since previous check returns None, this should as well, changed